scripts/netContain/scripts/ovsInit.sh

```diff
@@ -27,6 +28,11 @@ ovsdb-server --remote=punix:/var/run/openvswitch/db.sock --remote=db:Open_vSwitc
 echo "Starting ovs-vswitchd"
 ovs-vswitchd -v --pidfile --detach --log-file=/var/contiv/log/ovs-vswitchd.log -vconsole:err -vsyslog:info -vfile:info &
 
+while [[ $(ovsdb-client list-dbs | grep -c Open_vSwitch) -eq 0 ]] ; do
+    echo "Wait for ovsdb started"
```

Comment:
Can we change this to "Waiting for ovsdb to start"?

Also, should we fail after N attempts?  We do that in other places where we poll for readiness

Reply:
done